src/Roave/EmailTemplates/Service/TemplateService.php

```diff
@@ -123,7 +128,8 @@ public function __construct(
      */
     public function render($templateId, $locale, array $parameters = array())
     {
-        $template = $this->repository->getByIdAndLocale($templateId, $locale);
+        $template   = $this->repository->getByIdAndLocale($templateId, $locale);
+        $parameters = array_merge($this->options->getPredefinedParams(), $parameters);
```

Comment:
Please don't overwrite the variable - change its name so that the semantics are clear